packages/next-on-pages/src/fetch-handler/index.ts

```diff
@@ -0,0 +1,5 @@
+export default {
+	async fetch() {
+		return new Response();
+	},
+} as { fetch: ExportedHandlerFetchHandler<{ ASSETS: Fetcher }> };
```

Comment:
I think I'd also include this: 

```ts
import 'server-only';
```

because although we unfortunately can't make the build fail when the function is used throughout the app incorrectly, we can at least make sure that this is not shipped to the browser, to reduce the likeliness of end users seeing this error